.travis.yml

```diff
@@ -30,10 +30,8 @@ before_script:
   #- apt-get install graphicsmagick # for e2e screenshots
 
 script:
-  - npm run check-template
   - npm run test:single-run
   - npm run test:e2e
-  - npm run build:backwards
```

Comment:
what about the build?

Reply:
Already fixed by https://github.com/AnSyn/ansyn/pull/1304/commits/1fa8b602a22fa99bc26866372e8f3ee8393aa3f6